app/controllers/chefs/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Chefs::IngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:id])
+    @ingredients = @chef.ingredients.distinct
```

Comment:
This works, however I'd rather see this as a method in the Chef model, something like `all_ingredients` that way in the view, you could just call `@chef.all_ingredients`. When possible, we want to limit the number of ivars we're sending to the views.